### ISSUE TITLE
2.4.1 dependancy issue

### ISSUE THREAD

**Issue by davebaol** [evant/gradle-retrolambda#84]
Yesterday it was working fine, but now I get
````
      > Could not find any version that matches com.android.tools.build:gradle:0.8+.
        Searched in the following locations:
            https://jcenter.bintray.com/com/android/tools/build/gradle/maven-metadata.xml
        Required by:
            MultiProject:core:1.0 > me.tatarka:gradle-retrolambda:2.4.1
````
Actually, the link `http://mvnrepository.com/artifact/com.android.tools.build/gradle/0.8+` is broken.
BTW, I've taken that link from `http://mvnrepository.com/artifact/me.tatarka/gradle-retrolambda/2.4.1`, see 'Depends on' and click '0.8+'
Any idea what's happened?

**Comment by evant** [comment]
Could you post your gralde.build scripts?
Have you tried changing jcenter() to mavenCentral() in your buildscript repositories block?

**Comment by evant** [comment]
Hm, if you compare https://jcenter.bintray.com/com/android/tools/build/gradle/maven-metadata.xml
with https://repo.maven.apache.org/maven2/com/android/tools/build/gradle/maven-metadata.xml you'll see that a whole bunch of versions are missing from the `<versions>` section. Could that have something to do with it?

**Comment by davebaol** [comment]
Well, it looks like adding mavenCentral() before jcenter() did the trick.
Anyways, I didn't change my build script in the last week and it was working until today.

**Comment by evant** [comment]
I suspect there was a recent update to that `maven-metadata.xml` file that broke it. Unfortunately, I can't go back in time to compare. The plugin specifies a dependency on version `0.8+`, so gradle looks in that metadata file for a matching version. Since only `0.3`, `0.13.2` and `1.0.1` are listed, it doesn't find any that matches and fails. I _could_ release a new version that depends on 1.0.0 since that's the proper stable version anyway, but versions really should not be removed from that list. I'm not sure if this is an issue with jcenter itself or how google uploaded the dependency.

**Comment by evant** [comment]
I found that someone else reported this on the android issue tracker with regards to some other plugin. Linking here to track https://code.google.com/p/android/issues/detail?can=2&start=0&num=100&q=jcenter&colspec=ID%20Type%20Status%20Owner%20Summary%20Stars&groupby=&sort=&id=111912

**Comment by davebaol** [comment]
I think we can close this. Clearly it's not your responsibility :)

**Comment by evant** [comment]
I'm going to repoen this. It looks like google won't fix it on their end so I'll have to release a new version depending on 1.0.+

**Comment by davebaol** [comment]
Well, the answer of that google project member is somewhat stupid imo, because he says that it's dangerous to depend on pre-1.0 plugin, but the metadata xml doesn't contain1.0.0 while 2 older versions are there.
Also metadata at jcenter and mavenCentral are inconsistent, but still....

**Comment by evant** [comment]
Totally agree it's dumb, and I left a comment there of that effect, but I still should still update the dependency regardless.

**Comment by evant** [comment]
This has been fixed.
